### DISCUSSION TITLE
kitty 35 acting differently in Hyprland

### DISCUSSION THREAD

**Discussion by psi4j** [kovidgoyal/kitty#7484] [Q&A] [answered]
Latest kitty is suddenly flashing clear and then renders the background color in Hyprland on Wayland. For some reason when I downgrade the package on my Arch Linux build, it stops doing this. It is also the only application that is doing this on Hyprland, so I am confused as to why it's doing it.

What has changed from the last version to this one that could cause this that I might be able to look into to troubleshoot this? I already looked into this on the Hyprland side of things because I figured it would more than likely be the compositor, but it does it no matter what I change in hyprland.conf.

I didn't want to mark this as an issue because I haven't confirmed that this is a kitty issue, it might just as likely be some strange interaction kitty is having with NVIDIA and Hyprland. I haven't heard anyone talk about this, so I thought I'd ask around. Any insight or pointers in the right direction would be greatly appreciated. Thank you.

**Comment by kovidgoyal** [comment, marked as answer]
Dont have an NVIDIA card so cant help you with that. I can tell you its
working fine in Hyprland with non-NVIDIA cards. Generally speaking
wlroots based compositors + NVIDIA are quite unstable. I suggest you
either run kitty using XWayland or bisect to find whatever it is that is
causing the problem.

**Comment by psi4j** [comment]
Thank you, I will look into this. I'll post if I find any solutions in case anyone else encounters this.

**Comment by psi4j** [comment]
I've confirmed that kitty 35 is not having any issues with Hyprland on NVIDIA. It seems to be an issue with the particular build for version 35 from pacman. I don't know what was done differently, but when I build kitty myself, it doesn't have this issue. It actually runs even cleaner now with no artifacting when resizing incrementally using Hyprlands keybinds.
